src/assets/ts/main.ts

```diff
@@ -122,8 +122,18 @@ export function getWebAsset(assetId: string | null, user: User) {
   )
 }
 
-export function getAssetDashboardLink(assetId: string) {
-  return `https://login.screenlyapp.com/login?next=/manage/assets/${assetId}`;
+export function getTeamInfo(user: User, teamId: string) {
+  const queryParams = `id=eq.${encodeURIComponent(teamId || '')}`;
+  return callApi(
+    'GET',
+    `https://api.screenlyapp.com/api/v4.1/teams/?${queryParams}`,
+    null,
+    user.token
+  )
+}
+
+export function getAssetDashboardLink(assetId: string, teamDomain: string) {
+  return `https://${teamDomain}.screenlyapp.com/manage/assets/${assetId}`;
```

Comment:
@rusko124, this works when the user is logged in in the same browser session.

However, if a user isn't logged in (via the Screenly web interface), clicking the "View Asset" will open a login screen, and logging in will redirect you to a different domain name (for example, `domain-a.screenly.app.com` insteaf of `domain-b.screenlyapp.com`).

Is there a way to redirect to a specific Screenly team domain using `login.screenlyapp.com`?

Reply:
No, it will log in to the last team